### ISSUE TITLE
Are you still working on this project?

### ISSUE THREAD

**Issue by ian** [adam-cowley/neode#136]
Trying to figure out if it's dead or not.  Lots of issues reported and no responses.

**Comment by adam-cowley** [comment]
Yeah, still active.  I'm planning the next version the next version, just need to find the time to work on it
